### PR TITLE
Misc logging fixes for v1.3

### PR DESCRIPTION
**Pull request by joestringer**
Quieten and improve some warning/error messages. For more details, see individual commit logs.

The final commit, `endpoint: Only scrub CT when map already exists` is a little more than a logging change but the focus is the same. That commit may need a bit more attention.

Fixes: #5516 
Fixes: #5677
Fixes: #5679

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5680)

### CODE REVIEW

**Comment by joestringer** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

File: pkg/endpoint/bpf.go [hunk 1]
```diff
@@ -387,7 +387,14 @@ func (e *Endpoint) regenerateBPF(owner Owner, currentDir, nextDir string, regenC
 	// pre-existing connections using that IP are now invalid.
 	if !e.ctCleaned {
 		go func() {
-			e.scrubIPsInConntrackTable()
+			ipv4 := !option.Config.IPv4Disabled
```
Comment (rlenglet) [inline pkg/endpoint/bpf.go:390]: That doesn't look like "logging fixes" to me. Move those changes into a different PR?

**Review by rlenglet — CHANGES_REQUESTED** [review]
The logging fixes LGTM. Please move the ctmap fixes into a separate PR.

**Review by rlenglet — APPROVED** [review]
Nevermind. The commit messages explain why this is done. Thanks!

**Review by vadorovsky — APPROVED** [review]

**Review by vadorovsky — CHANGES_REQUESTED** [review]
Test failure seems to be related to your PR:

```
23:48:15 # github.com/cilium/cilium/pkg/datapath/loader
23:48:15 pkg/datapath/loader/loader_test.go:104:27: cannot use ep (type *testEP) as type endpoint in argument to compileAndLoad:
23:48:15 	*testEP does not implement endpoint (wrong type for Logger method)
23:48:15 		have Logger() *logrus.Entry
23:48:15 		want Logger(string) *logrus.Entry
23:48:15 pkg/datapath/loader/loader_test.go:123:27: cannot use ep (type *testEP) as type endpoint in argument to compileDatapath:
23:48:15 	*testEP does not implement endpoint (wrong type for Logger method)
23:48:15 		have Logger() *logrus.Entry
23:48:15 		want Logger(string) *logrus.Entry
23:48:15 FAIL	github.com/cilium/cilium/pkg/datapath/loader [build failed]
23:48:15 make: *** [tests-ginkgo-real] Error 1
23:48:15 Makefile:50: recipe for target 'tests-ginkgo-real' failed
23:48:16 Makefile:36: recipe for target 'tests-ginkgo' failed
23:48:16 make: *** [tests-ginkgo] Error 2
```

**Comment by joestringer** [comment]
test-me-please

File: pkg/endpoint/bpf.go (hunk 1 above)
Comment (joestringer) [reply pkg/endpoint/bpf.go:390]: Yeah, arguably that final commit could be in another PR. I can call it out in the description.

**Review by ianvernon — APPROVED** [review]
